modules/phase_field/tests/phase_field_kernels/tests

```diff
@@ -60,4 +60,11 @@
     input = 'ConstructionOrder.i'
     exodiff = 'ConstructionOrder_out.e'
   [../]
+
+  [./SimpleSplitCHWRes]
+    type = 'Exodiff'
+    input = 'SimpleSplitCHWRes.i'
+    exodiff = 'SimpleSplitCHWRes_out.e'
+    max_parallel = 1                
```

Comment:
Yes, exactly. Does that mean it can run in parallel, and we need to fix max_parallel to 1?